configure.ac

```diff
@@ -1194,6 +1194,7 @@ fi
 )
 if test "x$enable_prof" = "x1" ; then
   backtrace_method=""
+  AC_CHECK_LIB([kvm], [kvm_getprocs], [JE_APPEND_VS(LIBS, -lkvm)], [have_kvm=0])
```

Comment:
Is the result used anywhere, or should we add a guard based on this?

Reply:
I thought it was needed but not really finally.